reference/supervisor.md

```diff
@@ -770,6 +748,35 @@ overwrite silently existing files, so that the corresponding data may be lost.
 In the other APIs, the `Robot.saveWorld()` function can be called without
 argument. In this case, a simple save operation is performed.
 
+The `wb_supervisor_reload_world()` function sends a request to the simulator
+process, asking it to reload the current world immediately. As a result of
+reloading the current world, the supervisor process and all the robot processes
+are terminated and restarted. You may wish to save some data in a file from your
+supervisor program in order to reload it when the supervisor controller
+restarts.
+
+---
+
+**Name**
+
+**wb\_supervisor\_simulation\_reset** - *reset the simulation*
+
+{[C++](cpp-api.md#cpp_supervisor)}, {[Java](java-api.md#java_supervisor)}, {[Python](python-api.md#python_supervisor)}, {[Matlab](matlab-api.md#matlab_supervisor)}, {[ROS](ros-api.md)}
+
+```c
+#include <webots/supervisor.h>
+
+void wb_supervisor_simulation_reset();
+```
+
+**Description**
+
+The `wb_supervisor_simulation_reset()` function sends a request to the simulator process, asking it to reset the simulation immediately.
```

Comment:
This description is exactly the same as the reload world one.
As a user that will read the documentation I won't be able to understand the difference between these two functions.
The only explained difference is that the seed of random number generators is reset.